### ISSUE TITLE
"Comparison to average values of products in the same category" is unclear

### ISSUE THREAD

**Issue by zigouras** [openfoodfacts/openfoodfacts-server#2497]
**Describe the bug**
In the nutrients table, it is not clear what the object of the percentage is in the column of the products in the same category.  e.g. if it says +4%, does it mean that the category is 4% more than the product we are viewing or is the product 4% more than the category?  I think it is the latter but I was never sure!  Thanks.

**To Reproduce**
Look at any product.

**Expected behavior**
Some help text to clarify.

**Comment by stephanegigandet** [comment]
I agree, it's very unclear, but I'm not sure how to convey it in a visually pleasing way. Any suggestion?

**Comment by zigouras** [comment]
Just add some explanatory text, like "Calculated as average category value/this product value x 100%" e.g. below "Please note: for each nutrient, the average is computed for products for which the nutrient quantity is known, not on all products of the category."

**Comment by VaiTon** [comment]
> Or append the text to "Please note: for each nutrient, the average is computed for products for which the nutrient quantity is known, not on all products of the category."

Or maybe just a tooltip when hovering the value.

**Comment by YashasviChaurasia** [comment]
@teolemon I would like to work on this issue

**Comment by github-throwaway** [comment]
@YashasviChaurasia are you still working on this issue?

**Comment by YashasviChaurasia** [comment]
No, I am not working with this issue

**Comment by github-throwaway** [comment]
@zigouras is this issue still relevant?

**Comment by vedanth5** [comment]
Hi, I’d like to work on this issue. Could you please assign it to me?

**Comment by subhwastaken** [comment]
I found two places that can make this clearer: the gettext label for the percentage column and the product/category value explanation used in the nutrition comparison details. I updated the column wording to say the percentage is from the category average and added a short note that the two values shown are this product's value and the average for products in the same category.
